### ISSUE TITLE
Handle subscription to multiple items

### ISSUE THREAD

**Issue by anna-kartynnik** [codeforsanjose/gov-agenda-notifier#101]
Like here

<img width="893" alt="Screen Shot 2021-01-12 at 9 37 36 PM" src="https://user-images.githubusercontent.com/15210801/104411023-711c6c00-551e-11eb-8306-55dc7605f4ef.png">

https://xd.adobe.com/view/8a8cb36a-caa8-4bdf-b256-af108b9f911f-dfdf/screen/1a0b650d-ffe5-4656-8331-888700d83e1b/specs/

**Comment by anna-kartynnik** [comment]
@teohrt @sent-hil I have a question about the backend. I was told that the server supports creation of multiple subscriptions but I can't find the right mutation. I can see only `createSubscription` and it accepts the parameters for one subscription object. Am I missing anything or is it expected that I should perform one request per each subscription (several parallel requests to create several subscriptions)? If the latter, then it's not a good idea to have several simultaneous requests since it's possible that a user selects a lot of items to subscribe at once and we don't want to load the server by these multiple requests. I would suggest having a new mutation `createSubscriptions` (or modify the existing one) to accept an array of parameters to create a bunch of subscriptions. Another option is to limit the number of items that a user can select, but I'm not sure it's user-friendly 🤔...

**Comment by sent-hil** [comment]
@anna-kartynnik I can't find it either. I agree about creating a new mutation. What do you think about this as args:

```
{
  phone_number: '',
  email_address: '',
  meetings: [
    {
      meeting_item_id: '',
      meeting_id: ''
    }
  ]
}
```

This way I can loop over `meetings` and create a subscriptions.

**Comment by anna-kartynnik** [comment]
I think they look good!

**Comment by sent-hil** [comment]
@anna-kartynnik great, I will work on implementing it, should be pretty easy; should we deprecate `createSubscription`?

**Comment by anna-kartynnik** [comment]
I'm fine with deprecating `createSubscription`.

**Comment by anna-kartynnik** [comment]
Reopening since the frontend part is still in progress

**Comment by anna-kartynnik** [comment]
Implemented in PR #139.
